### PR TITLE
SONAR-7903 add missing primary keys to tables ce_task_input and scanner_context

### PR DESCRIPTION
**Pull request by simonbrandhof**

### CODE REVIEW

File: server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1302_create_table_ce_task_input.rb [hunk 1]
```diff
@@ -21,15 +21,15 @@
 #
 # SonarQube 6.1
 #
-class CreateTableCeTaskData < ActiveRecord::Migration
+class CreateTableCeTaskInput < ActiveRecord::Migration
```
Comment (sns-seb) [inline server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1302_create_table_ce_task_input.rb]: now that it's been deployed to Dory, indeed we can merge the two migrations
we couldn't before :)

Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1302_create_table_ce_task_input.rb]: yes I know :)

**Comment by sns-seb** [comment]
this is not fixing an existing DB. So Dory will keep on not having a PK on these tables. (same applies to developer local Databases).
Is is expected?

**Comment by simonbrandhof** [comment]
Yes it is expected. I will send a request to ops team for fixing Dory, and to developers to trash their local dbs.

**Comment by sns-seb** [comment]
then LGTM

**Pull request closed without merging**